src/main/java/uk/gov/hmcts/reform/pip/account/management/service/PublicationService.java

```diff
@@ -60,6 +60,30 @@ public boolean sendNotificationEmail(String emailAddress, String forename, Strin
         }
     }
 
+    /**
+     * Method which sends a request to the publication-services microservice to send a media account rejection email.
+     *
+     * @param mediaApplication - MediaApplication object containing applicant details
+     * @param reasons          - reasons for rejection
+     * @return boolean for logging success or failure
+     */
+    public boolean sendMediaAccountRejectionEmail(MediaApplication mediaApplication, String reasons) {
+        JSONObject jsonObject = new JSONObject();
```

Comment:
Could we use a model for this instead of JSONObject. Helps ensure type safety between the API's.

Reply:
Discussed and will be covering this as part of a seperate ticket, as this is also done elseware in this class